airflow/providers/amazon/aws/hooks/base_aws.py

```diff
@@ -484,6 +509,34 @@ def expand_role(self, role: str) -> str:
         else:
             return self.get_client_type("iam").get_role(RoleName=role)["Role"]["Arn"]
 
+    @staticmethod
+    def retry(fun: Callable):
+        """
+        A decorator that provides a mechanism to repeat requests in response to exceeding a temporary quote
+        limit.
+        """
+
+        def decorator_f(self):
+            quota_retry = getattr(self, 'quota_retry', None)
```

Comment:
Not sure about this parameter name -- it seems very specific to EcsOperator only.

Reply:
Good point.  Hm I've changed it to `retry_args` to be more generic, but I'm open to any suggestions.